docs/blog/arcade/update-jan-2023.md

```diff
@@ -0,0 +1,100 @@
+
+![Arcade game hardware consoles](/static/blog/arcade/update-jan-2023/arcade-machines.png)
+
+## Play Song
+
+More cowbell!!  We’ve gotten a lot of feedback that audio tracks and music in general is a super important part of any game (after all, what would Minecraft be without the soothing background music?). We’ve been itching to add a more detailed music editor that goes a little farther than our current 8-note Play Melody block. So, here it is – our v1 music editor, the **Play Song** block.
+ 
+![New play song block](/static/blog/arcade/update-jan-2023/play-song-block.png)
+
+![Music editor to create songs](/static/blog/arcade/update-jan-2023/music-editor.png)
+
+Watch this short demonstration video for an overview:
+
+https://youtu.be/7d0noM0BM7E
+
+<br/>
+
+This feature is very new, so over the next several months, we hope to work with all of you to refine the interface. We also need a gallery of songs, so if you’ve made any and are willing to share, please do!
+
+Head over to the [Forum](https://forum.makecode.com/c/share-your-arcade-projects-here/show-tell/13) and share your songs with us!
+
+## Color Palette
+
+When we were working on the [Wakanda Forever](https://arcade.makecode.com/#tutorial:/tutorials/wakanda-forever) tutorial, we realized the need for more custom palette colors for our games – especially to represent the diversity of skin tones for human characters in games. And we also wanted to just better help people express their creativity in general. So, in the **Asset Editor** we’ve added the ability to select from a variety of different color palettes, or you can make your own! Keep in mind that the color palette you use will apply to all art assets in your game, you are still limited to 15 colors, and the custom color palette you choose will only apply to that game/project.
+ 
+![Color palette editing](/static/blog/arcade/update-jan-2023/color-palette.gif)
+
+## Code Validation
+
+Being able to automatically validate the correctness of code in tutorials has been one of the top requested features from educators. In this release, we’ve added some basic code validation capabilities that will check for certain blocks on the workspace.
+
+![Code validation checks](/static/blog/arcade/update-jan-2023/code-validation.gif)
+
+For tutorial authors who would like to add code validation in their tutorials, please see the documentation on how to do this here <insert link to code validation documentation>
+
+To see code validation in action, check out these tutorials <link to tutorials>
```

Comment:
... and here?

Reply:
Yes, here we can say "To see code validation in action, check out these tutorials:
- [Time Flies](http://arcade.makecode.com/#tutorial:/tutorials/froggy)
- [Collect the Clovers](http://arcade.makecode.com/#tutorial:/tutorials/collect-the-clovers)